src/segment-loader.js

```diff
@@ -1176,31 +1169,33 @@ export default class SegmentLoader extends videojs.EventTarget {
       return null;
     }
 
+    let segmentMediaIndex = null;
+    let startOfSegment;
+    let syncRequest = false;
+
     // When the syncPoint is null, there is no way of determining a good
     // conservative segment index to fetch from
     // The best thing to do here is to get the kind of sync-point data by
     // making a request
     if (syncPoint === null) {
```

Comment:
Made this one big if/else so that we can log all of the data used to generate segment info at the end.